src/go/rdctl/cmd/snapshotList.go

```diff
@@ -81,10 +81,10 @@ func tabularOutput(snapshots []snapshot.Snapshot) error {
 		return nil
 	}
 	writer := tabwriter.NewWriter(os.Stdout, 0, 4, 4, ' ', 0)
-	fmt.Fprintf(writer, "NAME\tCREATED\n")
+	fmt.Fprintf(writer, "NAME\tDESCRIPTION\tCREATED\n")
```

Comment:
The `description` field can be quite long, can't it? Maybe it would be better to make it the rightmost field, and to only print the first part of it (up to maybe 60 characters?). And maybe to remove anything after the first newline?